docs/installation/native-installation.md

```diff
@@ -36,7 +36,17 @@ cd project-base
 - *The `--keep-vcs` option initializes GIT repository in your project folder that is needed for diff commands of the application build and keeps the GIT history of `shopsys/project-base`.*
 - *The `--stability=alpha` option enables you to install the project from the last alpha release. Default value for the option is `stable` but there is no stable release yet.*
 
-### 2. Install dependencies and configure parameters
+### 2. Set up configuration for native installation
+- Monolog is configured for logging into streams in `app/config/config.yml`.
+If you want to log into a file, change the configuration of handlers. For example 
+
+`cron`:
+    
+    `type`: rotating_file 
+    
+    `path`: "%kernel.logs_dir%/%kernel.environment%.log" 
```

Comment:
The formatting seems broken, also it's not clear where exactly to put the config...

I would try:
```markdown
```yml
monolog:
  # ...
  handlers:
    # ...
    cron:
      type: rotating_file
      max_files: 7
      path: "%kernel.logs_dir%/%kernel.environment%.cron.log"
` ` ` (without the spaces... haven't managed to escape it...)
```

Which will look like:

```yml
monolog:
  # ...
  handlers:
    # ...
    cron:
      type: rotating_file
      max_files: 7
      path: "%kernel.logs_dir%/%kernel.environment%.cron.log"
```